dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/EventQueryParams.java

```diff
@@ -693,8 +695,12 @@ public boolean orgUnitFieldIsValid()
      */
     public boolean fallbackCoordinateFieldIsValid()
     {
+        List<String> fcFields = fallbackCoordinateField == null ? new ArrayList<>()
+            : List.of( fallbackCoordinateField.split( "," ) );
```

Comment:
I see we split and join fallbackCoordinateField on "," several times in this PR. Wouldn't it make sense to hold it directly as a Collection instead?